src/pages/workspaces/workspace/notebooks/TerminalLauncher.js

```diff
@@ -101,10 +101,18 @@ const TerminalLauncher = _.flow(
       })
     } else {
       return div({ style: { padding: '2rem' } }, [
-        spinner({ style: { color: colors.primary(), marginRight: '0.5rem' } }),
-        (clusterStatus === 'Creating' || !cluster) ?
-          'Creating runtime environment. You can navigate away and return in 5-10 minutes.' :
-          'Starting runtime environment, this may take up to 2 minutes.'
+        (clusterStatus !== 'Stopped') && spinner({ style: { color: colors.primary(), marginRight: '0.5rem' } }),
+        (() => {
```

Comment:
I think this would read nicely as a `Utils.cond`. Doing so would eliminate the need to create a function and immediately invoke it. Let's pair on this. Tag me in anytime.

Reply:
Ah, I thought this felt familiar. Isaac made this comment already as a general comment which is why I didn't notice it when I made my comment. Anyway, still happy to pair on this.